src/main/kotlin/gg/essential/elementa/markdown/drawables/ParagraphDrawable.kt

```diff
@@ -161,7 +161,7 @@ class ParagraphDrawable(
                 if (targetWidth <= widthRemaining) {
                     // We can just layout this text drawable inline, next to the last one
                     layout(target, targetWidth)
-                    if (widthRemaining <= 0)
+                    if (widthRemaining < 0)
```

Comment:
I had forgotten to come back to this change and investigate further. If widthRemaining = 0, a new line will be started even if there's no text to draw on it. So, if the parent component's height is constrained to the markdown's height, the parent component ends up with an extra blank line at the bottom.

Would removing this check have an impact? Looking at it, it seems like it would start a new line anyway if the text needed to be split due to being wider than widthRemaining, but I'm not sure if there's a case that wouldn't apply to.

Reply:
I think I've found a solution that works assuming there isn't a case I've missed: https://github.com/EssentialGG/Elementa/pull/85/commits/fe8c402b94e486e4b11fabd406105349c5e121fb